.codespellrc

```diff
@@ -5,4 +5,5 @@ skip = .git,*.pdf,*.svg,msvc_strings.pyx,cython.st,test_genericclass_exttype.pyx
 ignore-regex= Tripple|/Fo|ist das|TOi|respect to the includee
 # forin - ? tag used in 3 files. may be could be renamed?
 # delimeters - typo, but used in API so kept as is
-ignore-words-list = asend,forin,nd,mis,te,nin,ue,ccompiler,ket,defin,delimeters,filetests,crasher
+# implementor,implementors - was decided that both forms are ok
```

Comment:
FWIW, I don't make "decisions" here, I just noted that both are valid English. It's up to the maintainers if they want to also eliminate rare(r) spelling variants.